packages/terra-compact-interactive-list/src/proptypes/columnShape.js

```diff
@@ -5,22 +5,44 @@ const columnShape = PropTypes.shape({
    * Required string representing a unique identifier for the column.
    */
   id: PropTypes.string.isRequired,
+
   /**
-   * String of text to render within the column header cell.
+   * String of text which will be used to provide a column context and to serve as a column header if needed.
    */
   displayName: PropTypes.string,
+
   /**
-   * Number that specifies the minimum column width in pixels.
+   * Width of the column in width units set by widthUnits prop, defaults to px.
+   * If not set, the column will be considered a flex growing column.
+   * If width is set, but flexGrow prop is also set to true, the width will be used as a flex basis of flex growing column.
    */
-  minimumWidth: PropTypes.number,
+  width: PropTypes.number,
+
+  /**
+   * Minimum width of the column in width units set by widthUnits prop, defaults to px.
+   * Will be disregarded if the width is set and flexGrow is omitted or false.
+   * Is reccomended in cases the column is flex growing column.
+   */
+  mimimumWidth: PropTypes.number,
+
   /**
-   * Number that specifies the maximum column width in pixels.
+   * Maximum width of the column in width units set by widthUnits prop, defaults to px.
+   * Will be disregarded if the width is set and flexGrow is omitted or false.
+   * Is reccomended in cases the column is flex growing column.
    */
   maximumWidth: PropTypes.number,
+
   /**
-   * A number (in px) specifying the width of the column. If not provided, the Table's default column width will be used.
+   * Whether the column width can grow (a flex growing column).
+   * Defaults to false if the width is set and true if the width is not set.
    */
-  width: PropTypes.number,
+  flexGrow: PropTypes.bool,
```

Comment:
What is the use case for the flex growing column?  I thought the consumers were specifying the widths?

Reply:
We have both, the `width` for cases when the consumers want to specify the exact width and `flexGrow` for cases when the column can grow. I understand it's confusing as here it looks like `width` was removed, but it is actually still there on line 19. Not sure why it's shown as removed here.